configure

```diff
@@ -173,7 +173,7 @@ fi
 printf "Verifying Qt build environment ... "
 
 if [ -z "$QC_QTSELECT" ]; then
-	QC_QTSELECT="${QT_SELECT/qt/}"
+	QC_QTSELECT="$(echo $QT_SELECT | sed "s/qt//")"
```

Comment:
That won't work, tr does something different, it removes all occurences of the characters 'q' and 't' and not only the substring "qt".

`> echo $(echo testqt  | tr -d "qt")`
`es`